pyproject.toml

```diff
@@ -86,13 +86,13 @@ docs = [
   "docutils >= 0.20.1",
   "jinja2 >= 3.1.4",
   "nbsphinx >= 0.9.4",
-  "nox >= 2024.4.15",
+  "nox >= 2024.10.9",
   "numpydoc >= 1.7.0",
   "pillow >= 10.4.0",
-  "pygments >= 2.18.0",
+  "pygments == 2.18.0",
```

Comment:
You made a comment in the the `.github/content/update-requirements-pr-bocy.md` that a PR should be opened to remove an upper bound on a dependency if one is add.  **Should this also be true if we freeze `==` a dependency?**

Reply:
Or maybe we only do this for install requirements (i.e. not tests, docs, etc.).

Reply:
> **Should this also be true if we freeze `==` a dependency?**

Yes! I'll need to update that.

 - [ ] address this ⬆️